zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/PublishNotification.java

```diff
@@ -1,101 +1,123 @@
                         .data("websiteHost", host.toString())
                         .data("eventType", eventType.name())
-                        .log("sending publish notification to website host");
-                try {
-                    Endpoint endpoint = new Endpoint(host, getEndPointName(eventType));
-                    Response<WebsiteResponse> response = http.postJson(endpoint, payload, WebsiteResponse.class);
-                    String responseMessage = response.body == null ? response.statusLine.getReasonPhrase() : response.body.getMessage();
-                    if (response.statusLine.getStatusCode() > 302) {
-                        info().data("websiteHost", host.toString())
-                                .data("responseMessage", responseMessage)
-                                .data("collectionId", payload.collectionId)
-                                .log("Error response from website for publish notification");
-                    } else {
-                        info().data("websiteHost", host.toString())
-                                .data("responseMessage", responseMessage)
-                                .data("collectionId", payload.collectionId)
-                                .log("Response from website for publish notification");
-                    }
-                } catch (Exception e) {
-                    error().data("collectionId", payload.collectionId)
-                            .data("websiteHost", host.toString())
-                            .data("eventType", eventType)
-                            .logException(e, "failed sending publish notification to website");
-                    String eventName = "";
-                    try {
-                        eventName = getEndPointName(eventType);
-                    } catch (BadRequestException ex) {
-                        eventName = "unknown: " + eventType.toString();
-                    }
-                    // FIXME it might be better to use collectionAlarm rather than alarm
-                    // but the NotificationPayload only has the collection ID
-                    SlackNotification.alarm(
-                            "Failed sending publish notifications to website",
-                            new PostMessageField("Event", eventName, true),
-                            new PostMessageField("Host", host.toString(), true),
-                            new PostMessageField("Collection ID", payload.collectionId, true)
-                    );
+                        .log("sending publish notification(s) to website host");
+                doSendNotification(httpInstance, host, eventType);
+            }
+        }
+    }
+
+    private void doSendNotification(Http http, Host host, EventType eventType) {
+        try {
+            if (Configuration.isLegacyCacheAPIEnabled()) {
+                if (eventType.equals(EventType.PUBLISHED) || eventType.equals(EventType.UNLOCKED)) {
```

Comment:
So I guess that's what I'm not following - if the previous logic also updated the cache times on an APPROVED event, why are we now ignoring it?

Reply:
That's because the `APPROVED` notifications are the only ones that have a list of URIs. So the current implementation is getting the list of URIs from that notification. However, the publish date that comes with those notifications is not the one we want. The publish date that comes with those notifications is the approval date. That's why we just ignore them, and only care for the `PUBLISHED` and `UNLOCKED` event types. Because even though those events do not come with a list of URIs, the come with a `CollectionDescription` object. That object has a list of URIs. You can see that in action if you check the `LegacyCacheApiPayloadBuilder` code. This is also explained in Confluence, in a page called "Zebedee notifications" (I'm not sure if I'm allowed to post Confluence links in GitHub, but I can send you a link in Slack).